website/docs/index.html.markdown

```diff
@@ -246,6 +246,9 @@ for more information about connecting to alternate AWS endpoints or AWS compatib
   - [`aws_dx_private_virtual_interface` resource](/docs/providers/aws/r/dx_private_virtual_interface.html)
   - [`aws_dx_public_virtual_interface` resource](/docs/providers/aws/r/dx_public_virtual_interface.html)
   - [`aws_dx_transit_virtual_interface` resource](/docs/providers/aws/r/dx_transit_virtual_interface.html)
+  - [`aws_ebs_snapshot` data source](/docs/providers/aws/d/ebs_snapshot.html)
+  - [`aws_ebs_snapshot` resource](/docs/providers/aws/r/ebs_snapshot.html)    
+  - [`aws_ebs_snapshot_copy` resource](/docs/providers/aws/r/ebs_snapshot_copy.html)    
```

Comment:
Since EC2/EBS Snapshots do not contain an account ID from the provider instance, this not necessary to document here.

```suggestion
```